apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -22,4 +22,11 @@ export const siteRouter = router({
       const { id } = input
       return getNavBar(id)
     }),
+  setNotification: protectedProcedure
```

Comment:
is there also a way to fetch the current notification(if any) set on the site? getConfig doesn't seem to provide this functionality yet it looks like

Reply:
Will add it